src/imageio.cs

```diff
@@ -2356,4 +2368,28 @@ interface CGImageAuxiliaryDataInfo {
 		NSData Data { get; set; }
 		NSDictionary DataDescription { get; set; }
 	}
+
+	[Mac (10, 15), iOS (13, 0), TV (13, 0), Watch (6, 0)]
+	[Static]
+	[Internal]
+	interface CGImageAnimationOptionsKeys {
+		[Field ("kCGImageAnimationDelayTime")]
+		NSString DelayTimeKey { get; }
+
+		[Field ("kCGImageAnimationLoopCount")]
+		NSString LoopCountKey { get; }
+
+		[Field ("kCGImageAnimationStartIndex")]
+		NSString StartIndexKey { get; }
+	}
+
+	[Mac (10, 15), iOS (13, 0), TV (13, 0), Watch (6, 0)]
+	[StrongDictionary ("CGImageAnimationOptionsKeys")]
+	interface CGImageAnimationOptions {
+		double DelayTime { get; set; }
+
+		NSNumber LoopCount { get; set; }
```

Comment:
I looked for code in GitHub that uses `kCGImageAnimationLoopCount` and it is used as follows

```objective-c

@property (nonatomic) NSUInteger loopCount;
...
options[(NSString *)kCGImageAnimationLoopCount] = @(self.loopCount);
```

```suggestion
		nuint LoopCount { get; set; }
```

From our [docs](https://docs.microsoft.com/en-us/xamarin/cross-platform/macios/binding/binding-types-reference#strong-dictionary-types) `nuint -> NSUInteger stored in an NSNumber` so we will handle this for the user

Reply:
done! :)